### DISCUSSION TITLE
Tone of voice question

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#2439] [Q&A] [answered]
Hi Paste Team 👋🏼

My question is around tone of voice. I have read the section that refers to this within the documentation (https://paste.twilio.design/foundations/content/voice-and-tone/#tonal-qualities) but wanted to confirm. 

Currently I am in a discovery period of exploring copy options for showing our internal and external customers their Twilio balance. I am wondering if using the copy '**Your** Balance' or '**Your** Bill' is in keeping with your patterns. 

I have attached a screenshot here as an example (please ignore the UI, it is just a concept for the moment to illustrate some hierarchy / copy). 

![Screenshot 2022-05-31 at 16 33 30](https://user-images.githubusercontent.com/104831341/171212438-0444f319-3de7-4812-83d7-bdb280d4c3d9.png)

Any thoughts would be great, thank you in advance!

**Comment by aren-twilio** [comment, marked as answer]
Hi Amy! I think _Your_ works well in this context - especially when paired with some explanatory text, which will make it feel a bit more natural. Definitely is in keeping with our tone of voice. We've got a bit more basic info about I/you/we/us in the [Product Style Guide](https://paste.twilio.design/foundations/content/product-style-guide#i-you-we-us) - will look to add a bit more specific advice.

**Reply by amyodea** [reply]
Brilliant, thank you for getting back to me so quickly!
